README.md

```diff
@@ -1 +1,25 @@
-# tdr-xray-logging
\ No newline at end of file
+# tdr-xray-logging
+
+This repo contains the custom AWS Distro for open telemetry custom image. Currently this is used by the frontend to add logging data to AWS-Xray.
+
+## How it works
+
+Within the ```/.github/workflows/``` folder we have the following scripts
+- build.yml
+- deploy.yml
+
+#### build.yml
+
+This script uses [ecs.build.yml](https://github.com/nationalarchives/tdr-github-actions/blob/main/.github/workflows/ecs_build.yml) from tdr-github-actions repo passing it the repo name, image name and secrets.
+This is then used to build and push the custom docker image to the ECR repositority with the correct versioning.
+
+#### deploy.yml
+
+This script uses the [ecs.deploy.yml](https://github.com/nationalarchives/tdr-github-actions/blob/main/.github/workflows/ecs_deploy.yml) from tdr-github-actions repo passing it environment and version to deploy.
+This then pulls the relevant Open telemetry custom image from ECR and pushes it to the frontend environment.
```

Comment:
This needs re-wording, to something like this: `This pulls the relevant version of the custom image, tags it with the environment and pushes the tagged image to the ECR repository.`

Its the ECS task definition in the the relevant environment which pulls the image from ECR to create a running container.